### DISCUSSION TITLE
How to update schema when I add a new field in the mapped class

### DISCUSSION THREAD

**Discussion by ghost** [sqlalchemy/sqlalchemy#9977] [Usage Questions]
Alembic is too heavy for my task. I wonder if there is some simple procedure with sqlalchemy alone.

I am in search of a similar function to `Base.metadata.create_all` that not only creates the table but is also able to `alter` the table based on the current mapped class.

**Comment by lelit** [comment]
No, AFAIK SA alone is not enough, because it does not implement all the logic to detect differences and render their `ALTER` counterparts statements.
Of course you can do that manually, that is, write a set of scripts that basically execute those `ALTER` statements in the correct order, but SA will be of little help.
I wonder why you say "_Alembic is too heavy for my task_": sure, it's another tool to learn, but once you learn it's (pretty simple) operative workflow and "bootstrap" your project to use it, it covers simple migrations completely by its own, and lets you define/refine more complex ones.

**Reply by ghost** [reply]
Thanks. I partial experience with "Alembic" is I have to generate the up/down python scripts for the migration. In my case, I just want to use some tool to always keep the schema in database to be in sync with that in file.
